src/main/java/uk/gov/di/ipv/cri/common/library/persistence/item/SessionItem.java

```diff
@@ -154,6 +155,14 @@ public void setAttemptCount(int attemptCount) {
         this.attemptCount = attemptCount;
     }
 
+    public EvidenceRequest getEvidenceRequest() {
+        return evidenceRequest;
+    }
+
+    public void setEvidenceRequest(EvidenceRequest evidenceRequest) {
```

Comment:
If my understanding is correct - KBV uses SessionHandlerTS so `evidenceRequest` is already set for us, but do you think if we're starting to update the Java code to `getEvidenceRequest` we should also update the Java code to `setEvidenceRequest` when we `saveSession` for consistency? Or it that out of scope?

Reply:
Agreed offline that this is out of scope and could impact lime